### PR TITLE
Ac quant ch ip

### PR DESCRIPTION
**Pull request by Acribbs**
I have not been able to test the pipeline functionality because of an issue with finding my python 2 conda environment. @sebastian-luna-valero I assume this is because I have an old environment:
```
The stderr was: 
                                                   /ifs/projects/adam/pipeline_tests/genesets/data/test_2/test_peakcallingSEIDR.dir/ctmp47kfnsvr: line 29: /ifs/devel/adamc/py36-v1/conda-install/bin/activate: No such file or directory
                                                   
                                                   source /ifs/devel/adamc/py36-v1/conda-install/bin/activate cgat-py2 &&         macs2 predictd         --format BAM         --ifile peakcalling_bams.dir/hsTh1-Tbet-R1_filtered.bam         --outdir peakcalling_bams.dir/hsTh1-Tbet-R1_filtered_insertsize.tsv.dir         --verbose 2         --mfold 10 30 --gsize hs         >& peakcalling_bams.dir/hsTh1-Tbet-R1_filtered_insertsize.log
                                                   -----------------------------------------
```

### CODE REVIEW

**Comment by Acribbs** [comment]
@sebastian-luna-valero The pipeline can be merged as peakcalling tests are passing

File: CGATPipelines/pipeline_peakcalling/pipeline.ini [hunk 1]
```diff
@@ -127,7 +127,7 @@ options_replicateconsistency=
 #    lowqual - remove reads based on MAPQ with samtools -bQ 
 #    contigs - remove all reads mapping to certain contigs
 #    blacklist  -remove all reads falling within blacklist region specified in bed
-bamfilters=unmapped,secondary,unpaired,duplicates,lowqual,contigs,blacklist
+bamfilters=unmapped
```
Comment (sebastian-luna-valero) [inline CGATPipelines/pipeline_peakcalling/pipeline.ini]: Why the other filters need removing?

File: CGATPipelines/pipeline_peakcalling/pipeline.ini [hunk 2]
```diff
@@ -33,10 +33,10 @@ input=1
 #
 ################################################################
 [annotations]
-database=!?
+database=/ifs/mirror/annotations/hg19_ensembl75/csvdb
```
Comment (sebastian-luna-valero) [inline CGATPipelines/pipeline_peakcalling/pipeline.ini]: According to what we are discussing in other pull requests, don't we need to leave this empty for portability?

**Review by sebastian-luna-valero — COMMENTED** [review]
Thanks @Acribbs 

I confirm the Jenkins tests are passing but could you please answer my questions below?

**Comment by Acribbs** [comment]
Sorry, I modified the ini and committed the changes by error. Please revert those ini changes if you have time.

I.e. Other filters should not have been removed and please leave database empty.

Thanks,
Adam

Sent from my iPhone

On 25 Jan 2018, at 11:14, Sebastian Luna-Valero <notifications@github.com<mailto:notifications@github.com>> wrote:


@sebastian-luna-valero commented on this pull request.

Thanks @Acribbs<https://github.com/acribbs>

I confirm the Jenkins tests are passing but could you please answer my questions below?

________________________________

In CGATPipelines/pipeline_peakcalling/pipeline.ini<https://github.com/CGATOxford/CGATPipelines/pull/382#discussion_r163784594>:

> @@ -127,7 +127,7 @@ options_replicateconsistency=
 #    lowqual - remove reads based on MAPQ with samtools -bQ
 #    contigs - remove all reads mapping to certain contigs
 #    blacklist  -remove all reads falling within blacklist region specified in bed
-bamfilters=unmapped,secondary,unpaired,duplicates,lowqual,contigs,blacklist
+bamfilters=unmapped


Why the other filters need removing?

________________________________

In CGATPipelines/pipeline_peakcalling/pipeline.ini<https://github.com/CGATOxford/CGATPipelines/pull/382#discussion_r163784728>:

> @@ -33,10 +33,10 @@ input=1
 #
 ################################################################
 [annotations]
-database=!?
+database=/ifs/mirror/annotations/hg19_ensembl75/csvdb


According to what we are discussing in other pull requests, don't we need to leave this empty for portability?

**Comment by sebastian-luna-valero** [comment]
Thanks @Acribbs 

Reverted back, and tests passing in Jenkins.

**Pull request merged**